benchmarks/hough_transform/procedural_c#/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Drawing;
+
+namespace procedural_c_
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			var image = new Bitmap("benchmarks/hough_transform/Pentagon.png");
+
+			var thetaAxisSize = 640;
+			var rhoAxisSize = 480;
+
+			var (sinTable, cosTable) = createCosSinTables(thetaAxisSize);
+
+			var outputData = makeHoughSpaceData(cosTable, sinTable, image, thetaAxisSize, rhoAxisSize);
+
+			var sum = 0;
+			for (int x = 0; x < outputData.GetLength(0); x++)
+			{
+				for (int y = 0; y < outputData.GetLength(1); y++)
```

Comment:
Er det undersøgt, at det ikke er et performance problem at kalde .GetLength for hver iteration? Hvis det er, bør det nok kaldes inden løkken. Vi arbejder jo også med kendte dimensione

Reply:
```suggestion
                        (int width, int height) = (outputData.GetLength(0), outputData.GetLength(1));
			for (int x = 0; x < width; x++)
			{
				for (int y = 0; y < height; y++)
```